.maintain/chaostest/src/config/README.md

```diff
@@ -0,0 +1,33 @@
+chaostest CONFIG
```

Comment:
Ah, sorry, I meant to include a single documentation comment above the definition of `Config`. Sorry for the confusion. This is fine as well.